Sources/StreamVideoSwiftUI/CallView/CallView.swift

```diff
@@ -159,18 +159,11 @@ public struct CallView<Factory: ViewFactory>: View {
         viewFactory.makeVideoParticipantsView(
             viewModel: viewModel,
             availableSize: size,
-            onViewRendering: handleViewRendering(_:participant:),
```

Comment:
Is it possible to do this without breaking changes to the public API?

Reply:
I remove this one as it's not being used anywhere. We can definitely go ahead without this change but makes our API confusing as we are asking for something what we won't be using.